substrate/frame/child-bounties/src/lib.rs

```diff
@@ -207,8 +221,14 @@ pub mod pallet {
 
 	/// The description of each child-bounty.
 	#[pallet::storage]
-	pub type ChildBountyDescriptions<T: Config> =
```

Comment:
but we should also keep old storage item, and deprecate it. otherwise it does not make sense, still breaking change